### ISSUE TITLE
text can be selected across panes

### ISSUE THREAD

**Issue by deathtrip** [aaronjanse/3mux#123]
On 3mux 1.1.0 when selecting text using shift+mouse, it can be selected across vertical and horizontal splits.
Just like you can without any multiplexer. It makes proper selecting multiple lines of t ext impossible.

**Comment by aaronjanse** [comment]
Yep, this is an known issue, and I need to document it. The cause is that 3mux cannot control the selection of the terminal emulator. It could possibly use something like `xclip` to put text on the clipboard, but that wouldn't work when accessing 3mux over ssh.

I think the cleanest solution is to copy text to the clipboard after making a pane full-screen using <kbd>Alt+Shift+F</kbd>.
